src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj

```diff
@@ -19,5 +21,61 @@
+
+  <Target Name="BuildWBT">
+    <PropertyGroup>
+      <_ScriptExt Condition="'$(OS)' == 'Windows_NT'">.cmd</_ScriptExt>
+      <_ScriptExt Condition="'$(OS)' != 'Windows_NT'">.sh</_ScriptExt>
+      <_Dotnet>$(RepoRoot)dotnet$(_ScriptExt)</_Dotnet>
+    </PropertyGroup>
+
+    <Exec IgnoreExitCode="true" Command="$(_Dotnet) build $(MSBuildThisFileDirectory)../../../wasm/Wasm.Build.Tests/Wasm.Build.Tests.csproj -c $(Configuration) -t:Test -p:TargetOS=browser -p:TargetArchitecture=wasm -p:XUnitClassName=none $(BuildAdditionalArgs)" />
+  </Target>
+
+  <Target Name="BuildBlazorFrame" DependsOnTargets="BuildSampleInTree;BuildWBT">
+    <ItemGroup>
+        <OverrideFiles Include="$(MSBuildThisFileDirectory)../../../../../src/mono/wasm/Wasm.Build.Tests/data/WasmOverridePacks.targets" />
+        <OverrideFiles Include="$(MSBuildThisFileDirectory)../../../../../src/mono/wasm/Wasm.Build.Tests/data/Blazor.Directory.Build.targets" />
+        <OverrideDestFiles Include="$(MSBuildThisFileDirectory)../blazor-frame/WasmOverridePacks.targets" />
+        <OverrideDestFiles Include="$(MSBuildThisFileDirectory)../blazor-frame/Directory.Build.targets" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(OverrideFiles)" DestinationFiles="@(OverrideDestFiles)" UseSymbolicLinksIfPossible="true" OverwriteReadOnlyFiles="true" />
+
+    <PropertyGroup>
+      <NugetConfigContent>&lt;?xml version=&quot;1.0&quot; encoding=&quot;utf-8&quot;?&gt;
+&lt;configuration&gt;
+  &lt;!-- Don't use any higher level config files. --&gt;
+  &lt;fallbackPackageFolders&gt;
+    &lt;clear /&gt;
+  &lt;/fallbackPackageFolders&gt;
+  &lt;packageSources&gt;
+    &lt;clear /&gt;
+    &lt;add key=&quot;nuget-local&quot; value=&quot;$(ArtifactsDir)packages/Release/Shipping/&quot; /&gt;
+    &lt;add key=&quot;dotnet8&quot; value=&quot;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json&quot; /&gt;
+    &lt;add key=&quot;nuget.org&quot;  value=&quot;https://api.nuget.org/v3/index.json&quot; protocolVersion=&quot;3&quot; /&gt;
+  &lt;/packageSources&gt;
+    &lt;disabledPackageSources&gt;
+    &lt;clear /&gt;
+  &lt;/disabledPackageSources&gt;
+&lt;/configuration&gt;
+      </NugetConfigContent>
+    </PropertyGroup>
+
+    <WriteLinesToFile
+        File="$(MSBuildThisFileDirectory)../blazor-frame/nuget.config"
+        Overwrite="true"
+        Lines="$(NugetConfigContent)" />
+
+    <Exec EnvironmentVariables="MSBuildSDKsPath=;DOTNET_ROOT=$(ArtifactsDir)bin/dotnet-latest;PATH=$(ArtifactsDir)bin/dotnet-latest:$(PATH)" WorkingDirectory="$(MSBuildThisFileDirectory)../blazor-frame" Command="dotnet publish blazor.csproj -c $(Configuration) -p:WBTOverrideRuntimePack=true -p:TargetOS=browser -p:TargetArchitecture=wasm $(BuildAdditionalArgs)" />
+
+    <ItemGroup>
+        <BlazorSourceFiles Include="$(MSBuildThisFileDirectory)../blazor-frame/bin/Release/net8.0/publish/wwwroot/blazor-template/**/*.*"/>
```

Comment:
`net8.0` could that be some known variable ?
`Release` are there other things preventing `Debug` configuration ?

Reply:
I am keeping net8.0 around as targeting the net9.0 doesn't work yet AFAIK.

I fixed the configuration.